execution/sourcestorage.go

```diff
@@ -23,8 +23,13 @@ var inputStreamIDPrefix = []byte("$input$")
 // GetRawStreamID can be used to get a new StreamID without saving it.
 func GetRawStreamID() *StreamID {
 	id := ulid.MustNew(ulid.Timestamp(time.Now()), rand.Reader)
+	return NewStreamID(id.String())
+}
+
+// GetRawStreamID can be used to get a new StreamID without saving it.
```

Comment:
Is it on purpose that this comment appears above `GetRawStreamID` and `NewStreamID` and says exactly the same thing?

Reply:
no, copy-paste, thank